evap/static/scss/components/_grid.scss

```
@@ -77,19 +77,17 @@
         "name semester responsible participants result"
         / auto    6rem       12rem         7rem  11rem;
 
-    @include media-breakpoint-down(md) {
+    @include media-breakpoint-down(lg) {
         grid:
-            "name semester  responsible responsible"
-            "name semester participants      result"
-            / auto    3rem         7rem       10rem;
+            "name semester responsible participants result"
+            / auto    3rem       7rem         10rem;
```

```suggestion
            / auto    3rem        7rem        6rem    9rem;
```
there seems to be a value missing. what do you think of this configuration?

---

Yes, I must have forgotten to copy some lines there... Now it is the same configuration as before, except the boundaries